PayPalMessaging/src/main/java/com/braintreepayments/api/PayPalMessagingPageType.kt

```diff
@@ -0,0 +1,56 @@
+package com.braintreepayments.api
+
+import com.paypal.messages.config.PayPalMessagePageType
+
+/**
+ * Message location within an application
+ * Note: **This module is in beta. It's public API may change or be removed in future releases.**
+ */
+enum class PayPalMessagingPageType {
+
+    /**
+     * Home view
+     */
+    HOME,
+
+    /**
+     * Individual product details view
+     */
+    PRODUCT_DETAILS,
+
+    /**
+     * Product listing
+     */
+    PRODUCT_LISTING,
+
+    /**
+     * Shopping cart view
+     */
+    CART,
+
+    /**
+     * Popover shopping cart view that covers part of the view
+     */
+    MINI_CART,
+
+    /**
+     * Checkout view
+     */
+    CHECKOUT,
+
+    /**
+     * Search results
+     */
+    SEARCH_RESULTS;
+
+    internal val rawValue: PayPalMessagePageType
```

Comment:
Is this value to map to an internal `PayPalMessagePageType` type in the messaging SDK? If it is, what are your thoughts on renaming to something like `internalValue`? I wasn't quite sure what `rawValue` meant in this case.

Reply:
Yes, both work for me, I think `rawValue` was used since it is the most common in iOS, but if `internalValue` is more Android, it's fine for me

Reply:
`rawValue` was suggested by @sshropshire [here](https://github.com/braintree/braintree_android/pull/895#discussion_r1465292799). I am not tied to it, so if there is a better representation for this on Android we should use it!

Reply:
That suggestion was to remove a redundant prefix. I could see `internalValue` making more since on Android. Raw Value is more of a Swift enum concept brought over from iOS.

Reply:
Updated here: e5b283ddf75275e451fbd15034045484b8af9312